Makefile

```diff
@@ -14,7 +14,7 @@ DIR = $(shell pwd)
 
 ANT ?= ant
 ANT_OPTIONS =
-KBASE_COMMON_JAR = kbase/common/kbase-common-0.0.20.jar
+KBASE_COMMON_JAR = kbase/common/kbase-common-0.0.22.jar
```

Comment:
Test dependencies from https://github.com/kbase/kb_sdk/pull/191/files would be useful here